hwdb/70-mouse.hwdb

```diff
@@ -482,6 +482,12 @@ mouse:usb:v046dpc06b:name:Logitech G700 Laser Mouse:
 # Logitech G700 Laser Mouse (Wireless)
 mouse:usb:v046dpc531:name:Logitech USB Receiver:
  MOUSE_DPI=*1000@500 3800@500 500@1000 1500@1000 2000@1000
+ 
```

Comment:
This line has a trailing space, which causes the tests to fail. Please fix. (You can run the tests with `ninja -C build test`.)